### DISCUSSION TITLE
How can i display embedded redash dashboards in frontend via spring gateway?

### DISCUSSION THREAD

**Discussion by surajsrivathsa** [getredash/redash#6622] [Q&A]
Hello,

Currently we directly use public urls to embed dashboards in our frontend and it is working. However i want to add a gateway in between redash and frontend with spring gateway. I am able to route the requests to redash and use its public url, [redash_dashboard_url](https://redash.myorg.net/public/dashboards/api_key_of_dashboard?org_slug=default ). 

This url works just fine in browser however when i try it via gateway it stops at first step after getting initial html, but returns 200 OK code. is there any way to get fully rendered dashboard directly from redash or any other way to execute the javascripts and download css midway.

```
<!DOCTYPE html>
<html ng-app="app" ng-strict-di>
  <head lang="en">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta charset="UTF-8">
    <base href="/">
    <title>Redash</title>

    <link rel="icon" type="image/png" sizes="32x32"
      href="/static/images/favicon-32x32.png">
    <link rel="icon" type="image/png" sizes="96x96"
      href="/static/images/favicon-96x96.png">
    <link rel="icon" type="image/png" sizes="16x16"
      href="/static/images/favicon-16x16.png">
    <link href="/static/vendors~app.dfhb3894378ndn908.css" rel="stylesheet"><link
      href="/static/app.wuejhd8437y4r87f2398uy.css" rel="stylesheet"></head>

  <body ng-class="bodyClass">
    <section>
      <app-view></app-view>
    </section>
    <script type="text/javascript"
      src="/static/vendors~app.2345124nin.js"></script><script
      type="text/javascript" src="/static/app.123456789hjds.js"></script></body>
</html>
```

**Comment by guidopetri** [comment]
I don't think we support direct exports like that :s but it would be a nice addition imo.
